src/main/resources/api/api.yaml

```diff
@@ -253,6 +420,30 @@ definitions:
   #
   # Issues
   #
+  V1Issue:
+    type: object
+    properties:
```

Comment:
I would also think about IssueStandard that we  should pass for creation(generation) issue.
I prefer that it would be optional parameter (in the case of absent, we cold use it from requires subject one)

Reply:
Look at board photo for more details

Reply:
So.. I need to create definition of IssueStandart and add it in definition of Issue?

Reply:
Leave TODO comment in model description, in place where corresponding property should be defined.